src/sass/components/sitemap.scss

```diff
@@ -1,5 +1,5 @@
 .sitemap {
-	margin-top: 4rem;
+	margin-top: 1rem;
```

Comment:
@sirleech another spot for `AU-space`. I know, I'm pedantic.

Reply:
I'll do this this arvo!